app/models/user.rb

```diff
@@ -54,25 +62,28 @@ def set_default_role
     self.role ||= :user
   end
 
-  def confirm_piv_cac?(proposed_uuid)
-    x509_dn_uuid == proposed_uuid if proposed_uuid
+  def piv_cac_configuration
+    PivCacConfiguration.new(self)
   end
 
-  def piv_cac_enabled?
-    PivCacLoginOptionPolicy.new(self).enabled?
+  def auth_app_configuration
+    AuthAppConfiguration.new(self)
   end
 
-  def piv_cac_available?
-    PivCacLoginOptionPolicy.new(self).available?
+  def personal_key_configuration
+    PersonalKeyConfiguration.new(self)
```

Comment:
Could we move these out of the User model? One of the rules we try to follow is to not place business logic in the User model. Where you have things like `configuration = current_user.piv_cac_configuration`, why not do `configuration = PivCacConfiguration.new(current_user)`?

Reply:
because I want a somewhat consistent interface for mfa configurations